.ci/wercker.sh

```diff
@@ -276,6 +276,32 @@ no-exception-alot-of-projects)
   rm -rf contribution
   ;;
 
+imports-guava)
```

Comment:
name should tell what it do.
name should be "no-error-imports-guava"

Reply:
done